modin/config/test/test_envvars.py

```diff
@@ -16,6 +16,9 @@
 import modin.config as cfg
 from modin.config.envvars import EnvironmentVariable, _check_vars, ExactStr
 
+import pyhdk
```

Comment:
This file does not assume that there will be engines in the environment.

We need to either make mock tests, or transfer the tests to files specific to `hdk`.

Reply:
These tests are sanity tests for HDK, not for the logic in modin - so I don't think a mock is particularly useful. I hid the import under a try/catch, it should pass if `pyhdk` isn't in the env.